backend/core/src/seeder/seeds/ami-charts/default-ami-chart-san-jose.ts

```diff
@@ -3,13 +3,13 @@ import { CountyCode } from "../../../shared/types/county-code"
 
 export class AmiDefaultSanJose extends AmiChartDefaultSeed {
```

Comment:
Why are we keeping the AmiDefaultSanJose class and using Bay Area values?  Would it make more sense to create a separate AmiDefaultBayArea class and remove the ones that aren't needed anymore?  Same with San Mateo and Triton below.

Reply:
see above